mozilla_schema_generator/common_ping.py

```diff
@@ -41,41 +41,34 @@ def _update_env(self, schema):
             "type": "object",
             "additionalProperties": string,
         }
-        boolean = {"type": "boolean"}
 
         def with_description(dtype: dict, comment: str) -> dict:
             """Add a description to the types defined above."""
             return {**dtype, **dict(description=comment)}
 
-        active_addons = prepend_properties(("environment", "addons", "activeAddons")) \
-            + ("additionalProperties", "properties")
-
-        schema.set_schema_elem(
-            prepend_properties(("environment", "settings", "userPrefs")),
-            with_description(
-                string_map,
-                "User preferences - limited to an allowlist defined in `toolkit/components/telemetry/app/TelemetryEnvironment.jsm`",  # NOQA
-            ),
-        )
-        schema.set_schema_elem(
-                prepend_properties(("environment", "system", "os", "version")), string)
-        schema.set_schema_elem(
-                prepend_properties(("environment", "system", "os", "hasSuperfetch")), boolean)
-        schema.set_schema_elem(
-                prepend_properties(("environment", "system", "os", "hasPrefetch")), boolean)
```

Comment:
I have now removed these three environment.system.os changes. All 3 fields are now present in the "raw" schemas and I see no reason to inject them here anymore. This also allows the descriptions from the raw schemas to stay intact such that they are propagated to the BQ schemas.